JsonPatch/JsonPatch.cs

```diff
@@ -15,6 +16,14 @@ namespace Json.Patch;
 [JsonConverter(typeof(PatchJsonConverter))]
 public class JsonPatch : IEquatable<JsonPatch>
 {
+#if NET8_0_OR_GREATER
+	/// <summary>
+	/// A TypeInfoResolver that can be used for serializing <see cref="JsonPointer"/> objects. Add to your custom
+	/// JsonSerializerOptions's TypeInfoResolver or TypeInfoResolveChain.
+	/// </summary>
+	public static IJsonTypeInfoResolver JsonTypeResolver => PatchSerializerContext.Default;
```

Comment:
I called this TypeInfoResolver in JsonSchema. I don't have a strong preference, but we should be consistent.